SQL/Cleanup_patches/2023-04-12-add-missing-downloadpath.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'DownloadPath', 'Where files are downloaded', 1, 0, 'text', ID, 'Downloads', 4 FROM ConfigSettings WHERE Name="paths";
```

Comment:
This should already be there. There's no cleanup patch required. According to `git grep` it was added pre-14.10 and I don't see anywhere it was removed. It's also already in the default schema in `SQL/0000-00-03-ConfigTables.sql`.

SQL/Archive/Pre-14.10/2014-09-25-ConfigToDB.sql:INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent) SELECT 'DownloadPath', 'Where files are downloaded', 1, 0, 'text', ID FROM ConfigSettings WHERE Name="paths";